docs/installation/upgrade.md

```diff
@@ -220,6 +222,14 @@ This article describes how to proceed in order to convert your Citrix XenServer
 * Eject CDs from your VMs before upgrading [to avoid issues](https://xcp-ng.org/forum/topic/174/upgrade-from-xenserver-7-1-did-not-work): `xe vm-cd-eject --multiple`
 * **It is very important to make sure clustering is not enabled on your pool**. It's a functionality that relies on proprietary software and that is not available in XCP-ng, and having it enabled before the upgrade will lead to XAPI being unable to start due to unexpected data in the database. If it is enabled or you already upgraded, see [this comment](https://github.com/xcp-ng/xcp/issues/94#issuecomment-437838544).
 * If you already have an XCP-ng pool, do not try to add a slave running XenServer to it. You will get an "Incompatible License" error. Please upgrade the slave to XCP-ng first, then add it to your existing pool
+* Before proceeding, it is important to ensure that the `xapi` service
+  currently running on your host is compatible with the version provided by
+  XCP-ng. To verify this, open a terminal session on the host machine and execute
+  the command `rpm -qi xapi-core`. This command will display detailed information
+  about the installed `xapi-core` package, including its version number.
+  Compatibility is crucial because the XCP-ng installation ISO supports upgrading
+  XenServer hosts only if their xapi version is up to and including version 24. If
+  your xapi version exceeds this, the upgrade process may not proceed correctly.
```

Comment:
We likely want to convey that this is "at the date of this writing", and maybe some "check here for more uptodate information".  Or maybe we should have a script producing a warning page (or just a bit of js magic) to have a perpetually-uptodate text?

Reply:
Keeping the doc up to date is always tricky.

I don't see the point of writing “at the date of this writing”: we want the user to have something to check, so they can know they shouldn't try to update their pool. “at the date of this writing” may make them think this could have change recently.

Reply:
We'll likely need this for the whole lifetime of XS 8.4 and XCP-ng 8.3, so I don't think we need to specify that it's at the time of writing.

We should also tell them how to check what the current version of XAPI available in XCP-ng 8.3 is.

Plus we must tell them that after the migration from XenServer they must install all updates and reboot to get the latest packages available from XCP-ng.

Reply:
I wouldn't know how to do it, otherwise I would have done @ydirson's “bit of js magic” to display the last xapi version available in the updates

Reply:
As a user, you can check https://mirrors.xcp-ng.org/8/8.3/updates/Source/SPackages/

Reply:
Or https://koji.xcp-ng.org/builds?inherited=0&tagID=65&order=-build_id&prefix=x&latest=1

Reply:
None is very user-friendly, but I don't have a better option at that time

Reply:
I do hope that "XCP-ng installation ISO supports upgrading XenServer hosts only if their xapi version is up to and including version 24" will be very quickly outdated, this is what my comment was about :)

Reply:
Good point. I missed that part.